src/Modal/Footer/Footer.spec.tsx

```diff
@@ -0,0 +1,20 @@
+// REACT
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+
+// STYLED COMPONENTS
+/* tslint:disable no-import-side-effect*/
+import 'jest-styled-components';
+/* tslint:enable */
+
+// COMPONENT
+import { Modal } from '..';
```

Comment:
Are you getting a circular import because of this? I had to be verbose with my sub-component `index.ts` imports and use the component file instead.

Reply:
Nah, because it's in the spec file, which isn't imported by anything.